src/views/AuthorityView/AuthorityView.js

```diff
@@ -129,38 +161,102 @@ const AuthorityView = ({
     return marcSourceClone;
   };
 
+  const onSubmit = () => {
```

Comment:
Let's call it `onConfirmDelete` because `onSubmit` makes me think that we're submitting a form